### PR TITLE
IBMCloud-Powervs docs and pre-req update

### PR DESCRIPTION
**Pull request by Vaibhav-Nazare**
Changes:
1. Moving from centos 8-stream to centos 9-stream for qcow2ova image.
2. Adding additional steps to setup a DHCP server
3. Updating pre-req.sh for installing additional packages `protobuf-compiler` and `device-mapper-devel`

### CODE REVIEW

File: src/cloud-api-adaptor/ibmcloud-powervs/README.md [hunk 1]
```diff
@@ -30,27 +30,33 @@ pvsadm image qcow2ova --prep-template-default > image-prep.template
...
 ```
-yum install -y gcc gcc-c++ git make
+yum install -y gcc gcc-c++ git make wget perl
+wget https://github.com/mikefarah/yq/releases/download/v4.42.1/yq_linux_ppc64le
+chmod +x yq_linux_ppc64le && mv yq_linux_ppc64le /usr/local/bin/yq
 git clone -b main https://github.com/kata-containers/kata-containers.git
 git clone https://github.com/confidential-containers/cloud-api-adaptor.git
 cd cloud-api-adaptor/ibmcloud-powervs/image
-make build
+AGENT_POLICY=no SEALED_SECRET=no make build
```
Comment (stevenhorsman) [inline src/cloud-api-adaptor/ibmcloud-powervs/README.md:39]: Is there are reason that you are disabling AGENT_POLICY? This is a feature we are using in other cloud providers. The same will likely be true for SEALED_SECRET soon (though it isn't an option at the moment)

Comment (Vaibhav-Nazare) [reply src/cloud-api-adaptor/ibmcloud-powervs/README.md:39]: We haven't given a try at the moment with these features being enabled, will revert the change for now, once we test these features we can have some idea on it and will raise additional PR for these changes if required.

File: src/cloud-api-adaptor/ibmcloud-powervs/README.md [hunk 2]
```diff
@@ -30,27 +30,33 @@ pvsadm image qcow2ova --prep-template-default > image-prep.template
...
 Add the following snippet to `image-prep.template`
 ```
-yum install -y gcc gcc-c++ git make
+yum install -y gcc gcc-c++ git make wget perl
+wget https://github.com/mikefarah/yq/releases/download/v4.42.1/yq_linux_ppc64le
+chmod +x yq_linux_ppc64le && mv yq_linux_ppc64le /usr/local/bin/yq
 git clone -b main https://github.com/kata-containers/kata-containers.git
 git clone https://github.com/confidential-containers/cloud-api-adaptor.git
-cd cloud-api-adaptor/ibmcloud-powervs/image
+cd src/cloud-api-adaptor/ibmcloud-powervs/image
```
Comment (Amulyam24) [inline src/cloud-api-adaptor/ibmcloud-powervs/README.md:38]: shouldn't it be `cd cloud-api-adaptor/src/cloud-api-adaptor/ibmcloud-powervs/image`?

File: src/cloud-api-adaptor/ibmcloud-powervs/README.md [hunk 3]
```diff
@@ -30,27 +30,33 @@ pvsadm image qcow2ova --prep-template-default > image-prep.template
 
 Add the following snippet to `image-prep.template`
 ```
-yum install -y gcc gcc-c++ git make
+yum install -y gcc gcc-c++ git make wget perl
+wget https://github.com/mikefarah/yq/releases/download/v4.42.1/yq_linux_ppc64le
+chmod +x yq_linux_ppc64le && mv yq_linux_ppc64le /usr/local/bin/yq
 git clone -b main https://github.com/kata-containers/kata-containers.git
```
Comment (Amulyam24) [inline src/cloud-api-adaptor/ibmcloud-powervs/README.md:36]: do we need to clone the repo - as it is being taken care of [here](https://github.com/confidential-containers/cloud-api-adaptor/blame/db56a2d3b30ff7e42cc100e6616ef049cdcc03b8/src/cloud-api-adaptor/podvm/Makefile.inc#L140)?

**Review by Amulyam24 — COMMENTED** [review]
Thanks for the PR @Vaibhav-Nazare!

File: src/cloud-api-adaptor/ibmcloud-powervs/README.md (hunk 3 above)
Comment (Vaibhav-Nazare) [reply src/cloud-api-adaptor/ibmcloud-powervs/README.md:36]: Will remove it, as was picked from the recent changes

File: src/cloud-api-adaptor/ibmcloud-powervs/README.md (hunk 2 above)
Comment (Vaibhav-Nazare) [reply src/cloud-api-adaptor/ibmcloud-powervs/README.md:38]: Updating

**Review by stevenhorsman — COMMENTED** [review]
Can you fix up the commits please

**Pull request closed without merging**